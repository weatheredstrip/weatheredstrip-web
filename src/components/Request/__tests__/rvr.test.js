import React from 'react'
import { render, cleanup } from '@testing-library/react'
import Rvr from '../rvr.react'
import '@testing-library/jest-dom/extend-expect'
afterEach(cleanup)

it('Rvr renders properly', () => {
  const rvr = 'http://atm.navcanada.ca/images/iwv/CYMX.png'

  const { container } = render(<Rvr data={rvr} />)
  const renderedRvr = container.firstChild
  expect(renderedRvr).toMatchSnapshot()
})

it('empty Rvr renders null', () => {
  const rvr = null
  const { container } = render(<Rvr data={rvr} />)
  const renderedRvr = container.firstChild
  expect(renderedRvr).toBeNull()
})
