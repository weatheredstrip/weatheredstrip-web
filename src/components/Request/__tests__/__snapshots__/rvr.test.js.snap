// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rvr renders properly 1`] = `
<img
  alt="Aerodrome Chart"
  id="rvr"
  src="http://atm.navcanada.ca/images/iwv/CYMX.png"
/>
`;
